mmdet/models/roi_heads/mask_heads/fused_semantic_head.py

```diff
@@ -30,10 +30,12 @@ def __init__(self,
                  in_channels=256,
                  conv_out_channels=256,
                  num_classes=183,
-                 ignore_label=255,
```

Comment:
should not delete them immediately. We should keep them and add deprecated warning for this two argument for three month at least